projects/dsp-ui/src/lib/search/expert-search/expert-search.component.ts

```diff
@@ -123,4 +125,16 @@ CONSTRUCT {
         return query + offsetTemplate;
     }
 
+    /**
+     * Get the IRI base url without configured api protocol.
+     * The protocol in this case is always http
+     */
+    getIriBaseUrl(): string {
+        return (
+            ('http://' + this._dspApiConfig.apiHost) +
```

Comment:
Does the [Rule: quotemark](https://palantir.github.io/tslint/rules/quotemark/) apply here?

Reply:
1e2974f enforces quote character for string literals.

Reply:
Yes it's correct ther. But you should do a comment instead of updating the code by yourself. Thanks 😉

Reply:
> do a comment instead of updating the code by yourself.

Why is that?

Reply:
It could 

> > do a comment instead of updating the code by yourself.
> 
> Why is that?

It could end in merge conflicts. What happens when I saw the wrong quotermarks locally too? I will change the code locally, but then I can't push because of another commit from someone else. So, I have to pull first and this could end in a merge conflict on my machine. Not everyone should push to the same branch especially not reviewers (unless it has been mutually agreed). Otherwise you have to review your own code, which doesn't make sense.

Reply:
In fact concatenation here could be replaced by ES6 template literals ;)

Reply:
@mpro7 You're right. I copied from js-lib. This is why I also had double quotation marks (we have other rules there). We should decide all together if we want to use ES6 literals or the "old" way. For me — and this is my opinion — the "old" way is better readable.

Reply:
@kilchenmann it is actually fine with short concatenations, but I find it really useful and more readable, especially for many properties and multi-liners. I've felt the same at the beginning as you, but since started to use template literals, I really see it more clear. [Here is few pros of using those](https://stackoverflow.com/questions/27565056/es6-template-literals-vs-concatenated-strings), because it is not only about readable but also more flexible code.

However above is just small thing, what I could find a little bit uncomfortable is no one rules set for all FE repos. But we could work on that soon.